### PR TITLE
add chartist to awesome-tools

### LINKED ISSUE #3: Add Chartist.js?
Chartist.js:
* http://gionkunz.github.io/chartist-js/
* https://github.com/gionkunz/chartist-js

Inactive?
* last commit was in 2019: https://github.com/gionkunz/chartist-js/commits/develop
* last release was in 2014: https://github.com/gionkunz/chartist-js/releases

### PR DESCRIPTION
**Pull request by Arantiryo**
Resolves #3

### CODE REVIEW

File: copy/tools/chartist.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+title: Chartist
+description: Simple responsive SVG charts
+logo: chartist.svg
+developer: null
+based_on: null
+licenses:
+  - type: open-source
+    title: MIT
+    link: 'https://github.com/chartist-js/chartist/blob/main/LICENSE-MIT'
```
Comment (dangreen) [inline copy/tools/chartist.yml:9]: There are two licenses

File: copy/tools/chartist.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+types:
+  - charts
+renders:
+  - svg
+features:
+  easy-to-customize: 0
```
Comment (dangreen) [inline copy/tools/chartist.yml:15]: it's easy to customize 🙂

**Review by dangreen — CHANGES_REQUESTED** [review]

File: copy/tools/chartist.yml (hunk 1 above)
Comment (Arantiryo) [reply copy/tools/chartist.yml:9]: If I add another license, only the last one is displayed. Which one should it be, MIT or WTFPL?

Comment (dangreen) [reply copy/tools/chartist.yml:9]: wtfpl is more fun )

Comment (Arantiryo) [reply copy/tools/chartist.yml:9]: Fixed : )

File: copy/tools/chartist.yml (hunk 2 above)
Comment (Arantiryo) [reply copy/tools/chartist.yml]: I changed it to 1 which places it between other tools with similar number of GH stars, hope that's alright.

**Review by dangreen — APPROVED** [review]

**Comment by dangreen** [comment]
@igorlukanin 👋 Pls review

**Review by igorlukanin — APPROVED** [review]

**Comment by igorlukanin** [comment]
Great job @Arantiryo @dangreen!

**Pull request merged**